proposals/wac-ucr/index.bs

```diff
@@ -1013,6 +1013,16 @@ Unfortunately, anyone can setup an identity provider, so Carol would
 like to be able to recognize credentials issued from trustworthy
 identity providers.
 
+### Block access to agents ### {#uc-blockagents}
+
+A nonpartisan group provides a public annotation service, and allows any
+[=authenticated agent=] to share fact-checked information.
```

Comment:
Does it mean that any WebID can share as long as not blacklisted?

Reply:
While it's feasible to blacklist an e.g. youtube commenter (I assume), it's kinda of an irritating cat-and-mouse game as the annoyer changes ids and the annoyed response to new junk by blacklisting yet another id. I believe the use case would be better-served if it were hosted in a more controlled server. How bout something like:

> A bank gives grants permission to expunge client late fees and surcharges to all managers *except* human resources managers and administrative managers.

Reply:
I agree with @ericprud . The use case as is seems to require one to prove that one is not someone, a nearly impossible task - at least if one does not presupposing identity systems such as passports where each ID ought to be in one-to-one relation with a physical person.  An example that closes the world to an institution helps with that.
 
At the same time, it creates a focus on processes internal to a company - where there are many answers to the problem already. Solid's advantage is in presenting an answer to cooperation between organisations and individuals. So if one could develop @ericprud's use case to something more open that would be more in the spirit.

Perhaps you can do as follows: You could make your organisation a bit less open, but still open to all people in a specific profession world wide. Eg: any person who can prove that they have a degree in immunology. Then it is easier to drop bad apples.

Reply:
>Does it mean that any WebID can share as long as not blacklisted?

Yes. The container is public append-only but some WebIDs are prohibited.

>it's kinda of an irritating cat-and-mouse game as the annoyer changes ids and the annoyed response to new junk by blacklisting yet another id.

>The use case as is seems to require one to prove that one is not someone, a nearly impossible task - at least without presupposing identity systems such as passports where each ID ought to be in one-to-one relation with a physical person.

That's the kind of a problem that actually originates at authentication itself. Current understanding is that ephemeral clients are allowed in the ecosystem. And its consequences go beyond the use case here.

>I believe the use case would be better-served if it were hosted in a more controlled server.

Possibly but I think it is important to acknowledge the most basic scenario.

If public append-only access can be a valid authorization policy, we need to tackle it head on. It doesn't have to be blocking per se but https://www.w3.org/2001/tag/doc/ethical-web-principles/#community for example holds in that there has to be ways to allow healthy conversation to take place while making it possible for people to protect themselves. As you know, there are endless horror stories.

We in fact have Solid servers and deployments out of the box with an append-only policy on WebID inboxes. Similarly, email inboxes... which is public append-only until of course sender's email because of their email address, IP, domain or whatever is blocked/blacklisted/denied/disallowed from ever entering the system.

How blocking (or alike) is done is not the main issue right now. The links I've shared in the original post are sufficient to acknowledge this as is IMO.

>A bank gives grants permission to expunge client late fees and surcharges to all managers except human resources managers and administrative managers.

Is that covered by UCs under #conditional-control ? If not, I'm open to group similar cases.